python/pyarrow/tests/test_table.py

```diff
@@ -348,10 +349,26 @@ def test_table_roundtrip_to_pandas_empty_dataframe():
     table = pa.table(data)
     result = table.to_pandas()
 
+    # TODO the conversion results in a table with 0 rows if there is no index
+    # column
```

Comment:
But there is an index column, isn't there? I don't understand this comment.

Reply:
Will try to clarify the comment. The DataFrame has an index, but if it's a default 0,1,2,..n range index, we don't actually create a column on the arrow side, but only store that information in the metadata. And the way the current pandas->arrow conversion is implemented, is that we get a list of arrays from the dataframe, and then basically do `Table.from_arrays`. But so in that implementation, if there are no arrays, `Table.from_arrays` can't return a table with a certain length ( so `num_rows` > 0 while `num_cols == 0`)